### PR TITLE
Fix BoomerAMGCGRelaxWt

### PR DESCRIPTION
**Pull request by victorapm**
This PR implements a check on whether CF_marker exists before using it.

### CODE REVIEW

**Review by rfalgout — APPROVED** [review]
Looks good!  Let me know if the tux tests are clean and I'll contact Al before we merge this.  Thanks!

**Comment by victorapm** [comment]
They are clean, Rob! I'll hold off the merge until we get confirmation this solves AI's problems. Thanks!

**Pull request merged**
